specification/subscription/resource-manager/Microsoft.Subscription/preview/2018-02-21-preview/Subscription.json

```diff
@@ -0,0 +1,294 @@
+                  "description":"OK. The request has succeeded.",
+                  "schema":{  
+                     "$ref":"#/definitions/OperationListResult"
+                  }
+               }
+            }
+         }
+      },
+      "/providers/Microsoft.Subscription/subscriptionOperations/{operationId}":{  
+         "get":{  
+            "description":"Retrieves the status of the subscription definition POST operation. The URI of this API is returned in the Location field of the response header.",
+            "operationId":"getSubscriptionOperation",
+            "responses":{  
+               "202":{  
+                  "description":"Accepted. Subscription update is in progress.",
+                  "headers":{  
+                     "Location":{  
+                        "description":"The URL where the status of the asynchronous operation can be checked.",
+                        "type":"string"
+                     },
+                     "Retry-After":{  
+                        "description":"The amount of delay to use while the status of the operation is checked. The value is expressed in seconds.",
+                        "type":"string"
+                     }
+                  }
+               },
+               "200":{  
+                  "description":"Successful completion of the asynchronous operation",
+                  "schema":{  
+                     "$ref":"#/definitions/SubscriptionCreationResult"
+                  }
+               }
+            },
+            "parameters":[  
+               {  
+                  "$ref":"#/parameters/apiVersionParameter"
+               },
+               {  
+                  "name":"operationId",
+                  "in":"path",
+                  "description":"The operation ID, which can be found from the Location field in the generate recommendation response header.",
+                  "required":true,
+                  "type":"string",
+                  "format":"uuid"
+               }
+            ]
+         }
+      },
+      "/providers/Microsoft.Subscription/subscriptionOperations":{  
+         "get":{  
+            "description":"Lists all of the available pending Microsoft.Subscription API operations.",
+            "operationId":"getSubscriptionOperations",
+            "responses":{  
+               "200":{  
+                  "description":"OK. The request has succeeded.",
+                  "schema":{  
+                     "$ref":"#/definitions/SubscriptionOperationListResult"
+                  }
+               }
+            },
+            "parameters":[  
+               {  
+                  "$ref":"#/parameters/apiVersionParameter"
+               }
+            ]
+         }
+      },
+      "/{scope}/providers/Microsoft.Subscription/createSubscription":{  
+         "post":{  
+            "description":"Creates an Azure subscription",
+            "operationId":"subscriptionFactory_createSubscription",
```

Comment:
Looks like other swaggers use pascal casing... Can you update this to "SubscriptionFactory_CreateSubscription"?